hw3/01-characters.js

```diff
@@ -1,4 +1,58 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+// async function to fetch the data from a given url
+const fetchData = async function fetchDataFromApi(url) {
+    try {
+        const res = await fetch(url);
+        const data = await res.json();
+        return data;
+    }
+    catch {
+        console.error('Error', error);
+    }
```

Comment:
The catch block is missing a name for the exception.
I think this should be `catch (error)` based on the next line.

Reply:
Corrected this in a future commit